react/tips/common-mistakes-in-react/react-dependency-array-for-useeffect-ii.md

```diff
@@ -0,0 +1,82 @@
+---
+author: kapnobatai136
+  
+aspects:
+
+  - workout
+
+  - deep
+
+type: normal
+
+category: tip
+
+links:
+  - '[Use deep compare effect](https://github.com/kentcdodds/use-deep-compare-effect){website}'
+
+---
+
+# Dependency array for useEffect (Part 2)
+
+---
+## Content
+
+The first consists of moving the dependency array outside of the component, although in some cases this might not be possible.

```

Comment:
This feels like a strange way to start an insight...

The first what?

Remember, insights should be share-able independently too as much as possible. If they do refer to a concept from a previous insights, that should be somehow mentioned, probably by summarizing the concept once more.

@kapnobatai137 can you add a sentence before this one that makes the transition between these insights smoother?